### PR TITLE
Kinetic Accelerator Parts Addition

### PR DESCRIPTION
**Pull request by Snakebittenn**
Addition to the protolathe, that is.
Probably needs some proofreading on the material costs, I dunno how people come up with this shit.

Adds Rapid core power converter
Phoron core power converter
kinetic charging KA cell
phoron loading KA cell
uranium loading KA cell
mhydrogen loading KA cell
vented kinetic accelerator frame
ultra heavy kinetic accelerator frame
vaurca kinetic accelerator frame
burst fire kinetic accelerator frame
large kinetic accelerator frame
long kinetic accelerator frame

### CODE REVIEW

**Comment by Snakebittenn** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/modules/research/designs/protolathe/mining_designs.dm [hunk 1]
```diff
@@ -78,6 +108,27 @@
...
+
+/datum/design/item/mining/ka_celluranium
+	req_tech = list(TECH_MATERIAL = 5, TECH_ENGINEERING = 6, TECH_MAGNET = 5, TECH_POWER = 5, TECH_PHORON = 5)
+	materials = list(DEFAULT_WALL_MATERIAL = 6000, MATERIAL_GLASS = 4000, MATERIAL_SILVER = 4000, MATERIAL_GOLD = 2000, MATERIAL_PHORON = 3000)
+	build_path = /obj/item/custom_ka_upgrade/cells/kinetic_charging
+
+/datum/design/item/mining/ka_cellhydrogen
+	req_tech = list(TECH_MATERIAL = 5, TECH_ENGINEERING = 6, TECH_MAGNET = 5, TECH_POWER = 5, TECH_PHORON = 5)
+	materials = list(DEFAULT_WALL_MATERIAL = 6000, MATERIAL_GLASS = 4000, MATERIAL_SILVER = 4000, MATERIAL_GOLD = 2000, MATERIAL_PHORON = 2000)
+	build_path = /obj/item/custom_ka_upgrade/cells/kinetic_charging
```
Comment (fernerr) [inline code/modules/research/designs/protolathe/mining_designs.dm:129]: The build path for all of these are identical.

**Review by fernerr — CHANGES_REQUESTED** [review]

File: code/modules/research/designs/protolathe/mining_designs.dm (hunk 1 above)
Comment (Snakebittenn) [reply code/modules/research/designs/protolathe/mining_designs.dm:129]: whoops lmao

**Review by fernerr — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]
I do have concerns regarding making the more powerful frames (E/B) available for construction.
We might have to revert that if misused.

**Pull request merged**